standard/standard/requirements/requirements_class/RC_Global.adoc

```diff
@@ -0,0 +1,22 @@
+[[req_class_basic_ypr_sdu]]
```

Comment:
Given that the ID of this requirements class is `/req/basic_ypr/sdu`, let's name this file path `requirements/basic_ypr/sdu.adoc`.

Individual requirements can be: `requirements/basic_ypr/sdu/xxx.adoc`.

Reply:
@manuelfuenmayor please help implement this new path pattern for all `[requirement]`, `[requirements_class]`, `[conformance_class]` and `[abstract_test]` blocks.